tdvt/tdvt/exprtests/dateparse/expected.setup.date.dateparse.month.1.txt

```diff
@@ -0,0 +1,42 @@
+<results>
+  <test name='DATEPARSE(&apos;yyyy-MM-dd&apos;, &apos;1234-06-01&apos;)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.mysql].[TEMP(Test)(2040050501)(0)]</column>
+      </schema>
+      <tuple>
+        <value>#1234-05-25 00:00:00#</value>
```

Comment:
Can you explain why `<value>#1234-06-01 00:00:00#</value>` is changed to  `<value>#1234-05-25 00:00:00#</value>` in both of these updated expected files?

Reply:
There's a gap between Julian Calendar switched Gregorian Calendar before October 1582. And our dateparse will change everything from Julian Calendar to Gregorian Calendar (More details at https://tfs.tsi.lan/tfs/DefaultCollection/Tableau-Dev/_workitems/edit/1360201/). I've tried to figure it out but the general code looks good to me. We have 2 options, 1)make an exception for GBQ for now and documented to make the dateparse go with 2023.1; 2)spend more time on it and fix it now.

Reply:
Hey Ross, could you ship this one?